auth-service/build.gradle.kts

```diff
@@ -37,16 +37,50 @@ dependencies {
     implementation("io.ktor:ktor-serialization-kotlinx-json-jvm:$ktor_version")
     implementation("io.ktor:ktor-server-netty-jvm:$ktor_version")
     implementation("io.ktor:ktor-network-tls-certificates:$ktor_version")
-    implementation("io.ktor:ktor-server-auth-ldap:$ktor_version")
+    implementation("io.ktor:ktor-server-status-pages:$ktor_version")
+    implementation("io.ktor:ktor-server-status-pages-jvm:$ktor_version")
+
+    // HTML templating
     implementation("io.ktor:ktor-server-thymeleaf:$ktor_version")
     implementation("io.ktor:ktor-server-thymeleaf-jvm:$ktor_version")
+
+    // Logging
     implementation("ch.qos.logback:logback-classic:$logback_version")
-    implementation("net.logstash.logback:logstash-logback-encoder:7.4")
+    implementation("net.logstash.logback:logstash-logback-encoder:7.4") // Structured log encoder
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-slf4j:1.7.2")
 
+    // OpenSAML
     implementation("org.opensaml:opensaml-core:4.0.1")
     implementation("org.opensaml:opensaml-saml-impl:4.0.1")
 
+    // Database
+    implementation("org.postgresql:postgresql:42.6.0")
+    implementation("com.zaxxer:HikariCP:5.0.1") // Connection pool
+    implementation("org.flywaydb:flyway-core:9.20.1") // Migrations
+
     testImplementation("io.ktor:ktor-server-tests-jvm:$ktor_version")
     testImplementation("org.jetbrains.kotlin:kotlin-test-junit:$kotlin_version")
+    testImplementation("org.mockito.kotlin:mockito-kotlin:5.0.0")
+}
+
+// Migrations are run on startup outside of Development mode.
+// Extra task to run Flyway migrations without starting the
+task("migrate", JavaExec::class) {
+    mainClass.set("uk.gov.communities.delta.auth.services.DbPoolKt")
+    classpath = sourceSets["main"].runtimeClasspath
+}
+
+tasks {
+    "run"(JavaExec::class) {
```

Comment:
Oh, is this equivalent to `task("run", JavaExec::class) {` like you used above?

Reply:
Nope, that gives "Cannot add task 'run' as a task with that name already exists." 🤷